pkg/client/functions.go

```diff
@@ -23,9 +23,9 @@ import (
 type FunctionsClient interface {
 	// Function Runner
 	RunFunction(ctx context.Context, organizationID string, run *v1.Run) (*v1.Run, error)
-	GetFunctionRun(ctx context.Context, organizationID string, functionName string, runName string) (*v1.Run, error)
-	ListRuns(ctx context.Context, organizationID string) ([]v1.Run, error)
-	ListFunctionRuns(ctx context.Context, organizationID string, functionName string) ([]v1.Run, error)
+	GetFunctionRun(ctx context.Context, organizationID string, functionName string, runName string, since *int64) (*v1.Run, error)
```

Comment:
functions are growing in arguments, and I expect there will be more. It might be a good time to introduce `FunctionOpts` structure.